src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-python.mdx

```diff
@@ -40,47 +40,61 @@ import opentelemetryPythonLogCapturedinTraceSpan from 'images/opentelemetry_scre
 
 
 
-<SideBySide>
-  <Side>
-    Try out any of these Python tutorials to see what the New Relic platform can do with your OTLP data. We have three tutorials you can choose from, each one using the same demo Flask app. The app will calculate the nth number in the Fibonacci sequence and generate traces, metrics, and logs.
+Try out any of these Python tutorials to see what the New Relic platform can do with your OTLP data. We have three tutorials you can choose from, each one using the same demo Flask app. The app will calculate the nth number in the Fibonacci sequence and generate traces, metrics, and logs.
```

Comment:
A little nitpicky, but do we need to use future tense here? I think present tense works fine (kind of like how you refer to a book in present tense) and makes for a cleaner read